bindata/etcd/pod.yaml

```diff
@@ -129,6 +129,33 @@ ${COMPUTED_ENV_VARS}
 ${COMPUTED_ENV_VARS}
       - name: "ETCD_STATIC_POD_VERSION"
         value: "REVISION"
+  # The etcdbackup container copies snapshots from etcd data dir into backup volume.
+  - name: etcdbackup
+    image: ${IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/sh
+        set -euo pipefail        
+        cp --verbose --recursive --preserve --reflink=auto /var/lib/etcd/ /var/backup/etcd
```

Comment:
I suggest you make this a command, so we have some go code we can also test properly. Eg in here: https://github.com/openshift/cluster-etcd-operator/tree/master/pkg/cmd

Didn't we want to take the snapshot with etcdctl for starters? and there's no retention and schedule either